backend/app/controllers/api/v1/researcher/studies_controller.rb

```diff
@@ -33,19 +36,90 @@ def update
     inbound_binding, error = bind(params.require(:study), Api::V1::Bindings::StudyUpdate)
     render(json: error, status: error.status_code) and return if error
 
-    inbound_binding.update_model!(@study)
+    @study.update(inbound_binding.to_hash.except(:researchers, :stages))
+
+    notify_researchers(Array(inbound_binding.researchers)) unless inbound_binding.researchers.nil?
+
+    unless inbound_binding.stages.nil?
+      @study.stages.clear
+      inbound_binding.stages.each do |stage|
+        s = Stage.where(id: stage.id).find_or_create_by(stage.to_hash.merge({ config: {} }))
+        @study.stages << s
+      end
+    end
 
     response_binding = Api::V1::Bindings::Study.create_from_model(@study)
     render json: response_binding, status: :ok
   end
 
+  def update_status
+    if params[:study].present?
+      study_update, error = bind(params.require(:study), Api::V1::Bindings::StudyUpdate)
+      render(json: error, status: error.status_code) and return if error
+
+      @study.update(study_update.to_hash.except(:researchers, :stages))
+    end
+
+    if params[:status_action] == 'submit'
+      (survey_id, secret_key) = CloneSurvey.new.clone(@study.title_for_researchers)
+      @study.stages.each do |stage|
+        stage.update({
+          status: :waiting_period,
+          config: {
+            type: 'qualtrics',
+            survey_id: survey_id,
+            secret_key: secret_key
+          }
+        })
+      end
+      ResearcherNotifications.notify_kinetic_study_review(@study)
+    end
+
+    if params[:stage_id].present?
+      if params[:status_action] == 'pause'
+        
+      end
+
+      if params[:status_action] == 'resume'
+
+      end
+    end
+
+    if params[:status_action] == 'launch'
+      @study.stages.update_all(status: 'active')
+    end
+
+    render json: Api::V1::Bindings::Study.create_from_model(@study), status: :ok
+  end
```

Comment:
pretty sure robocop is going to complain about complexity of this method.  Maybe split the submit block and the other checks out into their own functions?